### PR TITLE
Node modules

### PR DESCRIPTION
**Pull request by willyaranda**

### CODE REVIEW

File: test/functions/common.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ exports.allDifferents = function allDifferents(l) {
   return true;
 };
 
-var DEBUG = false;
```
Comment (frsela) [inline test/functions/common.js]: I prefer this magic debug value to be false by default

File: src/common/logger.js [hunk 2]
```diff
@@ -61,7 +62,7 @@ logger.prototype = {
       return;
     }
 
-    var logmsg = "[" + this.appname + " # " + level + "] - {" + (new Date()) + " (" + Date.now() + ")} - " + message;
+    var logmsg = "[" + this.appname + " # " + level + "] - " + this.pid + " - {" + Date.now() + "} - " + message;
```
Comment (frsela) [inline src/common/logger.js:65]: cool this is really more useful :)

**Comment by willyaranda** [comment]
I'm doing a lot of tests with this PR, ignore it ;)

**Comment by frsela** [comment]
one nit.
please check travis

**Comment by willyaranda** [comment]
@frsela fixed. This fixes travis builds, update modules to latest versions. Good to merge.

**Pull request merged**